### PR TITLE
Add initial draft of PA copy

### LINKED ISSUE #398: [Spotlight] First pass at PA copy

### PR DESCRIPTION
**Pull request by macfarlandian**
## Description of the change

Adds generic baseline copy for all the sections expected to be in the initial PA release, with the expectation that some if it may be augmented with more specific copy later. This should make the pages all basically functional at the launch content scope — the charts are erroring until the next successful data export goes through, but I tested against pre-release data and all but one of the sections were working fine. The one section that isn't is parole success rates, which is a known issue. You also won't be able to filter to individual prisons or parole offices yet because the locality lists have not yet been finalized.

## Type of change

- [ ] Bug fix (non-breaking change that fixes an issue)
- [x] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Configuration change (adjusts configuration to achieve some end related to functionality, development, performance, or security)

## Related issues

> Closes #398 

### Development

These boxes should be checked by the submitter prior to merging:

- [x] Manual testing against realistic data has been performed locally

### Code review

These boxes should be checked by reviewers prior to merging:

- [x] This pull request has a descriptive title and information useful to a reviewer
- [x] This pull request has been moved out of a Draft state, has no "Work In Progress" label, and has assigned reviewers
- [x] Potential security implications or infrastructural changes have been considered, if relevant

### CODE REVIEW

File: spotlight-client/src/contentApi/sources/us_pa.ts [hunk 1]
```diff
@@ -20,39 +20,434 @@ import { TenantContent } from "../types";
...
+        body: `<p>
+          Disparities emerge long before a person is incarcerated. By the time someone
+          comes under the DOC’s care, they have been arrested, charged, convicted, and
+          sentenced. Even before contact with the criminal justice system, disparities
+          in community investment (education, housing, healthcare) may play an important
+          role in creating the disparities that we see in sentencing data.
+        </p>
+        <p>
+          {ethnonymCapitalized} make up {beforeCorrections.populationPctCurrent} of
+          North Dakota’s population, but {beforeCorrections.correctionsPctCurrent} of
```
Comment (daschi) [inline spotlight-client/src/contentApi/sources/us_pa.ts:341]: Another ND reference 👀

File: spotlight-client/src/contentApi/sources/us_pa.ts [hunk 2]
```diff
@@ -20,39 +20,434 @@ import { TenantContent } from "../types";
...
+        different state, but have a compelling reason to serve their parole in
+        Pennsylvania. Under the same program, individuals paroled from a Pennsylvania
+        prison may occasionally serve their parole in a different state.
+      </p>`,
+      totalLabel: "Total people on parole",
+    },
+    ParolePopulationHistorical: {
+      name: "Historical Parole Population",
+      methodology: `<p>
+        This data includes the number of people that were on parole in North Dakota on
```
Comment (daschi) [inline spotlight-client/src/contentApi/sources/us_pa.ts:185]: There's a leftover ND reference here

**Review by daschi — COMMENTED** [review]
Just a couple references to update

File: spotlight-client/src/contentApi/sources/us_pa.ts (hunk 1 above)
Comment (macfarlandian) [reply spotlight-client/src/contentApi/sources/us_pa.ts:341]: ahhhh good catch! 😐

**Review by daschi — APPROVED** [review]

**Pull request merged**